### PR TITLE
[docs] Add changelog v0.23.0

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
+![A report that contains a stack trace for a client JS error](docs/articles/images/client-error-stack-report.png)
+
+#### :gear: Browsers are Automatically Restarted When They Stop Responding ([#1815](https://github.com/DevExpress/testcafe/issues/1815))
+
+If a browser stops responding while it executes tests, TestCafe restarts the browser and reruns the current test in a new browser instance.
+If the same problem occurs with this test two more times, the test run finishes.
+
+#### :gear: Work Reliably in Harsh Network Conditions ([#2727](https://github.com/DevExpress/testcafe/issues/2727))
+
+TestCafe now works stably in networks where any request can fail unexpectedly.
```
Comment (kirovboris) [inline CHANGELOG.md]: Why we don't mention the `unstableNetworkMode` option?

Comment (kirovboris) [reply CHANGELOG.md]: Also can we define this option w/ CLI?

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+#### :gear: View the JavaScript Errors' Stack Traces in Reports ([#2043](https://github.com/DevExpress/testcafe/issues/2043))
+
+Now when a JavaScript error occurs on the tested webpage, the test run report includes a stack trace for this error (only if the [--skip-js-errors](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#-e---skip-js-errors) option is disabled).
+
+![A report that contains a stack trace for a client JS error](docs/articles/images/client-error-stack-report.png)
+
+#### :gear: Browsers are Automatically Restarted When They Stop Responding ([#1815](https://github.com/DevExpress/testcafe/issues/1815))
+
+If a browser stops responding while it executes tests, TestCafe restarts the browser and reruns the current test in a new browser instance.
+If the same problem occurs with this test two more times, the test run finishes.
```
[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: @AlexKamaev says test won't be reported as failed - everything just finishes

[the remark this answers is not in the document]
Comment (AlexKamaev) [reply CHANGELOG.md]: I mean that test won't be reported as failed, but error will occur and tests execution will stop

[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: yeah, and probably `the test run finishes` is a good diplomatic way to say about this.

[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: However, we can say `the test run finishes and an error is thrown`

[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply CHANGELOG.md]: Although, I don't think we should post a screenshot with this global error.

**Comment by VasilyStrelyaev** [comment]
Will https://github.com/DevExpress/testcafe/issues/2846 be included in `v.0.23.0`?

**Comment by AndreyBelym** [comment]
Yep, we should add #2846.

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**